packages/lab/src/__tests__/__e2e__/overlay/Overlay.cy.tsx

```diff
@@ -50,8 +50,9 @@ describe("GIVEN an Overlay", () => {
     });
 
     it("THEN it should trap focus within Overlay once opened", () => {
-      cy.mount(<Default open />);
```

Comment:
Is controlled version tested?

Reply:
Added tests to make sure the onOpenChange function is called correctly in a controlled state